enterprise/models.py

```diff
@@ -1132,6 +1132,9 @@ def save(self, *args, **kwargs):
 
         return super().save(*args, **kwargs)
 
+    def get(self, *args, **kwargs):
+        return super().select_related('profile').get(*args, **kwargs)
```

Comment:
Does this actually work? Because `profile` is related to an `auth.User`, not an `EnterpriseCustomerUser`.